src/sentry/models/project.py

```diff
@@ -98,6 +98,7 @@ class Project(Model):
         'pk',
         'slug',
     ])
+    platform = models.CharField(max_length=64, null=True)
```

Comment:
Ok great - thanks for the go ahead from ops side Matt and I'll switch to your suggestion Cramer and move the PUT to ProjectDetailsEndpoint